graphql_api/query/project.py

```diff
@@ -145,6 +153,21 @@ def resolve_label_distribution(
         auth.check_project_access(info, project_id)
         return manager.get_label_distribution(project_id, labeling_task_id, slice_id)
 
+    def resolve_confidence_distribution(
+        self,
+        info,
+        project_id: str,
+        labeling_task_id: str = None,
+        slice_id: str = None,
+        num_samples: int = 100,
```

Comment:
```suggestion
        labeling_task_id: Optional[str] = None,
        slice_id: Optional[str] = None,
        num_samples: int = 100,
```
typing :), may need to add at the top: from typing import Optional